int32 x_max
int32 x_min
int32 y_max
int32 y_min
BoundingBox boundingbox

